connect/src/spirc.rs

```diff
@@ -233,11 +234,11 @@ impl Spirc {
 
         let (cmd_tx, cmd_rx) = mpsc::unbounded();
 
-        let volume = config.volume as u16;
+        let volume = config.volume;
         let linear_volume = config.linear_volume;
 
         let device = initial_device_state(config, volume);
```

Comment:
If you make `SpircTask::set_volume` write to device you can remove the volume argument from this function.
Set the device volume (inside `initial_device_state`) with 0, and overwrite it later when you call `task.set_volume`